PolyEngine/Engine/Src/EnginePCH.hpp

```diff
@@ -7,8 +7,8 @@
 #include "Engine.hpp"
 
 // ECS
-#include "ComponentBase.hpp"
 #include "Entity.hpp"
+#include "ComponentBase.hpp"
```

Comment:
Then the problem is somwhere else. Entity.hpp problably includes something that is required by ComponentBase.